browser/resources/settings/brave_appearance_page/toolbar.html

```diff
@@ -46,6 +46,9 @@
     </settings-checkbox>
   </div>
 </template>
+<if expr="enable_sidebar">
+  <settings-brave-appearance-sidebar prefs="{{prefs}}"></settings-brave-appearance-sidebar>
```

Comment:
Please put the import for this in the sibling .ts file and remove from the brave_overrides/appearance_page.ts file

Reply:
Moved. Thanks for catching!